src/utils/cosmostation.ts

```diff
@@ -12,7 +12,7 @@ export const getCosmostationExtensionOfflineSigner = (
 ): OfflineSigner => ({
   getAccounts: async () => {
     const response = await window.cosmostation.cosmos.request({
-      method: 'cos_account',
+      method: 'cos_requestAccount',
```

Comment:
should we try catch the old code and only request when it throw?

Reply:
I tested, `cos_requestAccount` is basically the same UX if the account request has been approved